test/sagemaker_tests/tensorflow/inference/test/integration/sagemaker/test_tfs.py

```diff
@@ -160,3 +168,43 @@ def test_python_model_with_lib(boto_session, sagemaker_client,
     # python service adds this to tfs response
     assert output_data["python"] is True
     assert output_data["dummy_module"] == "0.1"
+
+
+@pytest.mark.model("resnet50")
+def test_resnet_with_inference_handler(
```

Comment:
did you test this on TF2.5?

Reply:
Good point. I tried it only once, but I didn't really use this same script. So I'll retry it to ensure that it works correctly.